src/FileFormats/CBF/write.jl

```diff
@@ -21,6 +21,7 @@ mutable struct _CBFDataStructure
     dcoord::Vector{Tuple{Int,Int,Int,Float64}}
     variables_with_domain::Set{MOI.VariableIndex}
     variable_cones::Vector{Tuple{Vector{MOI.VariableIndex},String}}
+    scalar_variables::Vector{Int}
```

Comment:
This is the identity except for exponential cones right ? Maybe a comment saying that would be helpful for readability